modules/tensor_mechanics/test/tests/capped_weak_plane/tests

```diff
@@ -110,6 +110,7 @@
     type = Exodiff
     input = 'pull_and_shear_1step.i'
     exodiff = 'pull_and_shear_1step.e'
+    abs_zero = 1e-7
```

Comment:
@bwspenc Can you write some requirements for these tests or do you want me to try to make something up?

Reply:
FYI, I am working on falcon now to see if I can get this solve to be a bit more robust.

Reply:
OK, I am done. I just made a comparison file. There were alot of variables that are zero being compared.

Reply:
I can take a stab at it.  I really don't know anything about this model, and there are a ton of tests in this file. Is this the last one?

Reply:
This should be the last one, this is actually a new one that showed up after changing the default environment on the build machine. Thank you.

Reply:
@aeslaughter I added those SQA requirements in PR #12569, and @WilkAndy says they look OK. You can either merge that and wait for it to go in to devel, or cherry-pick that commit into your branch for this PR, whichever is easier for you. Hopefully we're almost there!